website/views/shared/tasks/task.jade

```diff
@@ -133,10 +139,10 @@ li(bindonce='list', bo-id='"task-"+task.id', ng-repeat='task in obj[list.type+"s
               span=env.t('addChecklist')
         form.checklist-form(ng-if='task.checklist')
           fieldset.option-group(ng-if='!$state.includes("options.social.challenges")')
-            legend.option-title
+            legend.option-title(ng-if='task.checklist[0]')
```

Comment:
If you'd like to experiment with "?", please do, but be aware that we might decide to stay with the current situation. Saving space is good, but providing easily-found hints to users is more important. Although having said that, I think a "?" icon could be an improvement! If we used it, we'd probably want it to be visible both with and without a checklist for consistency, so when a checklist existed, the "Checklist" text would become just a title and the "?" icon would be next to it too.